prisma/migrations/20250303170613_tags_on_image_new/migration.sql

```diff
@@ -0,0 +1,128 @@
+DROP VIEW IF EXISTS "TagsOnImageDetails";
+DROP TABLE IF EXISTS "TagsOnImageNew";
+
+CREATE TABLE "TagsOnImageNew" (
+  "imageId" integer NOT NULL,
+  "tagId" integer NOT NULL,
+  "attributes" smallint NOT NULL,
+  PRIMARY KEY ("imageId", "tagId")
+);
+
+-- CreateIndex
+CREATE INDEX "TagsOnImageNew_imageId_idx" ON "TagsOnImageNew"("imageId");
+CREATE INDEX "TagsOnImageNew_needsReview_idx" ON "TagsOnImageNew" ("attributes") WHERE ("attributes" & 1 << 09) != 0;
+CREATE INDEX "TagsOnImageNew_disabled_idx" ON "TagsOnImageNew" ("attributes") WHERE ("attributes" & 1 << 10) != 0;
+
```

Comment:
Do we want to index tagId? Is there ever a use-case where we search images for a specific tagId, if yes, when and how often would we need it?

Reply:
It looks like there are some use cases in the code for querying by tagId, though I'm not sure how many of them are still relevant. We could always add the index for now and drop it later if we find we no longer need it.